doc/classes/AABB.xml

```diff
@@ -182,6 +182,7 @@
 			<argument index="0" name="aabb" type="AABB">
 			</argument>
 			<description>
+				Returns true if this AABB and [code]aabb[/code] are approximately equal, by calling [code]is_equal_approx[/code] on each component.
```

Comment:
"true" should be formatted as inline code: `[code]true[/code]`.

Reply:
The reference to `is_equal_approx` could be a link to the method documentation: `[method @GDScript.is_equal_approx]` (btw its own documentation should also use code formatting for true/false).

Reply:
Done, but only `[method` etc for Vector2, Vector3, Color, Quat, and Plane, because the other structures don't use the method directly.